tests/cucumber/features/authentication/authentication.feature

```diff
@@ -7,3 +6,13 @@ Feature: Login
   Background:
     Given I am on the home page
     Given I am signed out
+
+  @dev
+  Scenario: Create account
+    Given "test@test.com" has not registered
```

Comment:
Yeah, you will need to make a call to a reset() function *before* running your tests. The mirrors database persists between test runs. That's probably why you're seeing that error. Because that test user really *does* exist. I think the my_fixtures.js file defines a reset() method. If not, that's where it should go.